src/docs/product/performance/transaction-summary.mdx

```diff
@@ -8,6 +8,20 @@ Every transaction has a summary view that gives you a better understanding of it
 
 ![Example of Transaction Summary](./perf-transaction-summary.png)
 
+## What Is a Transaction?
+
+A transaction represents a single instance of an activity you want to measure or track, like a page load, page navigation, or an asynchronous task. Having transaction information lets you monitor the overall performance of your application beyond when it crashes or generates an error. Without transactions, you can only know when things in your application have actually gone wrong — which is important, but not the whole picture.
+
+Transactions on the Performance page are grouped by the transaction names, such as an endpoint paths for backend request transactions (like `/store/checkout/` or `api/v2/users/<user_id>/`) or URLs/routes for page load transactions (like `https://docs.sentry.io/performance-monitoring/`). When you click a transaction on the **Performance** page, it takes you to the **Transaction Summary**, where you can see a list of transaction events: that is, the individual instances of that specific transaction.
```

Comment:
Is it more grammatically correct to say `such as an endpoint's path`?

Reply:
yep good catch!